Sources/App/add.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  remote.swift
+//  Guaka
+//
+//  Created by Omar Abdelhafith on 05/11/2016.
+//
+//
+
+import Guaka
+
+var addCommand: Command = Command(
+  name: "add", 
+  run: executeAddCommand, parent: rootCommand)
```

Comment:
All functions names are closures in swift. so run will work like that, it either accepts a function name, or a closure. (swift is coooul)